addon/models/parse-user.js

```diff
@@ -18,8 +18,8 @@ var ParseUser = DS.Model.extend({
 
 ParseUser.reopenClass({
   requestPasswordReset: function( email ) {
-    var adapter = this.get( 'store' ).adapterFor( this ),
```

Comment:
The idea behind using `this` was that you can use *any* model name as the parse user. You just extend this class.

I believe you can continue to pass `this` to `adapterFor`, or if that isn't possible without deprecation instead pass `this.modelName`.